Tools/VTuneProfiler/tachyon/sample.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "tachyon",
+  "description": "Simple ray tracer demonstrating the use of Intel&reg; oneTBB and OpenMP along with optimizations based on Intel VTune&trade; Profiler results.",
+  "categories": ["Toolkit/oneAPI Tools/VTune Profiler"],
+  "toolchain": ["dpcpp"],
+  "languages": [{"cpp":{}}],
+  "dependencies": ["tbb", "vtune"],
+  "targetDevice": ["CPU"],
+  "os": ["linux", "windows"],
+  "builder": ["ide", "cmake"],
+  "altindex": true,
+  "guid": "8C866569-DCEC-43D1-85EB-E725FBF18591",
+  "ciTests": {
+         "linux": [{
+                "steps": [
+                        "mkdir build",
+                        "cd build",
+                        "cmake ..",
+                        "make",
+                        "make run"

```

Comment:
@jenniferdimatteo Remove `"make run"`. Since this sample outputs a GUI, the CI won't be able to verify that the GUI works, only that the sample compiles successfully. Also, remove the "," after `make,` to ensure that it's a proper JSON file.